### DISCUSSION TITLE
Time axis in profile page can not show normal

### DISCUSSION THREAD

**Discussion by jiaohu** [community/community#37955] [Other Feature Feedback, Questions, & Ideas] [answered]
the time axis can not show normal  after I switch it several times.
<img width="235" alt="image" src="https://user-images.githubusercontent.com/13608671/199177227-58f63096-a243-43e5-a31c-f78756a76d97.png">
<img width="420" alt="image" src="https://user-images.githubusercontent.com/13608671/199177339-72caa69f-1299-49ea-bb95-8fb9f896cf6f.png">

**Comment by radar-blip** [comment, marked as answer]
That same thing happened a while back, about a month ago. 
You can file a bug-report if you want to make the team aware of it: [Bug-Report](https://support.github.com/contact/bug-report)
Most likely it will be fixed soon.

**Reply by jiaohu** [reply]
TKS👌

**Reply by radar-blip** [reply]
No problem.
Also, I think if you refresh the page it should go back to normal.

**Reply by jiaohu** [reply]
emmm, I have refreshed the page many times. It recurs stably.
